$NetBSD: patch-src_libstd_build.rs,v 1.3 2019/01/19 12:44:08 ryoon Exp $

SunOS support:
 - Support PKGSRC_USE_SSP (ugly for now).

Updated for rust 1.41

--- rustc-1.41.0-src/src/libstd/build.rs.orig   2020-06-11 15:09:47.516012445 +0000
+++ rustc-1.41.0-src/src/libstd/build.rs        2020-06-11 15:10:28.916564849 +0000
@@ -25,6 +25,9 @@
         println!("cargo:rustc-link-lib=posix4");
         println!("cargo:rustc-link-lib=pthread");
         println!("cargo:rustc-link-lib=resolv");
+        println!("cargo:rustc-link-lib=ssp");
+        // Use libumem for the (malloc-compatible) allocator
+        println!("cargo:rustc-link-lib=umem");
     } else if target.contains("apple-darwin") {
         println!("cargo:rustc-link-lib=System");
